docs/troubleshooting/error-while-attempting-to-call-load-module-data.md

```diff
@@ -0,0 +1,5 @@
+# Error While Attempting to Call 'load' or 'loadModuleData' Inside Holocron Module
```

Comment:
I wasn't sure what would be the best `Possible Ways to Fix It`

Reply:
this is usually an uncaught error in the `loadModuleData` promise chain that bubbles up and gets caught by One App. The way to debug it depends on the function you are executing there. suggesting adding try catches to debug the actual error maybe?